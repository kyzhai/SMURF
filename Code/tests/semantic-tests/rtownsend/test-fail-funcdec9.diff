1c1
< Fatal error: exception Sast.Main_missing
---
> Fatal error: exception Sast.Multiple_identical_pattern_lists("true _ x 1 m")
