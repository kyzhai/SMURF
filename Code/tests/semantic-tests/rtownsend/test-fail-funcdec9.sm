/* Pattern lists must be unique for multiple function declarations */
f :: Bool -> Int -> Int -> Int -> Int -> Int

f True _ g 1 a = 4
f True _ x 1 m = 6

main = []
