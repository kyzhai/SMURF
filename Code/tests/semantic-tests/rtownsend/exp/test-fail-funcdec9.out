Fatal error: exception Sast.Multiple_identical_pattern_lists("true _ x 1 m")
